goagen/codegen/workspace.go

```diff
@@ -310,7 +310,7 @@ func PackagePath(path string) (string, error) {
 		if gp, err := filepath.Abs(gopath); err == nil {
 			gopath = gp
 		}
-		if strings.HasPrefix(absPath, gopath) {
+		if filepath.HasPrefix(absPath, gopath) {
```

Comment:
Hmm I was looking at the wrong implementation, the [Windows version](https://golang.org/src/path/filepath/path_windows.go) does do the right thing. Still the comment indicates this should not be used. Not sure what the alternative is though.